runner/plugin.ts

```diff
@@ -43,40 +43,38 @@ export class Plugin {
   /**
    * @param {!Context} context The context that this plugin should be evaluated
    *     within.
-   * @param {function(*)} done
    */
-  execute(context: Context, done: (message?: string) => void): void {
+  async execute(context: Context): Promise<void> {
     try {
-      require(this.packageName)(
-            context, context.pluginOptions(this.name), this);
+      const plugin = require(this.packageName);
+      plugin(context, context.pluginOptions(this.name), this);
     } catch (error) {
-      return done('Failed to load plugin "' + this.name + '": ' + error);
+      throw `Failed to load plugin "${this.name}": ${error}`;
     }
-    done();
   };
 
   /**
    * Retrieves a plugin by shorthand or module name (loading it as necessary).
    *
    * @param {string} name
-   * @param {function(*, Plugin)} done
    */
-  static get(name: string, done: (err: any, plugin?: Plugin) => void): void {
+  static async get(name: string): Promise<Plugin> {
```

Comment:
Loading a plugin eventually becoming an asynchronous action seems even more likely to me.